Gardner.css

```diff
@@ -0,0 +1,386 @@
+}
+.max-width
+{
+    max-width: 1200px;
+    margin: auto;
+}
+.navbar .max-width
+{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.navbar.sticky
+{
+    padding: 15px 0;
+    background: #607d3b;
+}
+.navbar .menu li
+{
+    list-style: none;
+    display: inline-block;
+}
+.navbar .menu li a
+{
+    color: #000000;
+    font-size: 16px;
+    font-weight: 500;
+    margin-left: 25px;
+    transition: color 0.3s ease;
+}
+.navbar .menu li a:hover
+{
+    color: #709D61;
+}
+.menu-btn
+{
+    color: #000000;
+    font-size: 23px;
+    cursor: pointer;
+    display: none;
+}
+.home
+{
+    display: flex;
+    height: 100vh;
+    min-height: 400px;
+}
+.home .max-width
+{
+    margin-left: 174.4px;
+}
+.home .text-1
+{
+    font-size: 45px;
+    font-weight: bold;
+}
+.home .text-1 span
+{
+    color: #709D61;
+}
+.home-content
+{
+    width: 600px;
+    /* margin-left: 20px; */
+}
+.home .text-2
+{
+    font-size: 45px;
+    font-weight: bold;
+}
+.home .text-3
+{
+    font-size: 16px;
+    font-weight: normal;
+    color: #949494;
+    margin-top: 30px;
+}
+.homeimg
+{
+    float:right;
+    margin-top: 130px;
+    margin-right: 160px;
+    height: 450px;
+    width: 554px;
+}
+.home .home-content a
+{
+    display: inline-block;
+    background: #709D61;
+    color: #ffffff;
+    font-size: 16px;
+    padding: 12px 36px;
+    margin-top: 30px;
+    border-radius: 6px;
+    border: 2px solid #709D61;
+    transition: all 0.3s ease;
+
+}
+.home .home-content a:hover
+{
+    color: #000000;
+    background: none;
+}
+.about .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+    text-align: center;
+}
+.about .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   text-align: center;
+
+}
+.about-content
+{
+    margin-top: 10px;
+    display: flex;
+    text-align: center;
+    width: 1200px;
+}
+.plan
+{
+    font-weight: bold;
+    color: #3a3a3a;
+    width: 100%;
+    margin: auto;
+    font-size: 20px;
+    margin-top: 20px;
+}
+.plan-cont
+{
+    width: 90%;
+    color: #949494;
+    margin: auto;
+    margin-top: 20px;
+}
+/* welcome styling */
+.weltext
+{
+    float: right;
+    margin-top: 90px;
+}
+.welcome .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+}
+.welcome .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   width: 90%;
+   margin-bottom: 30px;
+}
+.welcome .weltext
+{
+    width: 50%;
+}
+.welcome .welcont
+{
+    width: 95%;
+    color: #949494;
+}
+.welbutton a
+{
+    display: inline-block;
+    background: #ffffff;
+    color: #709D61;
+    font-size: 16px;
+    padding: 12px 36px;
+    margin-top: 30px;
+    border-radius: 6px;
+    border: 2px solid #709D61;
+    transition: all 0.3s ease;
+    font-weight: bold;
+}
+.welbutton a:hover
+{
+    background: #709D61;
```

Comment:
Repeating hex values are supposed to be in a CSS variable

Reply:
Have changed to RGB values